src/admin_extra/service_admin_extra_accounts.erl

```diff
@@ -218,7 +218,8 @@ delete_old_users(Days, Users) ->
                       SecOlder :: non_neg_integer()) -> boolean().
 delete_old_user({LUser, LServer}, TimeStampNow, SecOlder) ->
     %% Check if the user is logged
-    case ejabberd_sm:get_user_resources(LUser, LServer) of
+    JID = jid:make(LUser, LServer, <<>>),
```

Comment:
In a situation like this, when we know for sure that `LUser` and `LServer` at hand are already stringprepped, why not call `jid:make_noprep/3` to avoid the cast prepping altogether?